lib/utils.js

```diff
@@ -258,6 +258,12 @@ function getModuleChunk(module, modules) {
   return null;
 }
 
```

Comment:
Oops totally missed this, will update.

May I suggest [inch-ci](https://inch-ci.org/) and a script like [this](https://github.com/obartra/ssim/blob/master/ci/doc.sh) to automate JSDoc validation as part of CI?